spec/rspec/rails/matchers/action_cable/have_broadcasted_to_spec.rb

```diff
@@ -226,5 +226,43 @@ def broadcast(stream, msg)
         end
       end
     end
+
+    it "has a description when not chained" do
+      expect(have_broadcasted_to("stream").description).to eq("have broadcasted exactly 1 messages to stream")
+    end
+
+    it "has a description when not chained and aliased" do
+      expect(broadcast_to("stream").description).to eq("broadcast exactly 1 messages to stream")
+    end
+
+    it "has a description when stream name is passed as an array" do
+      expect(have_broadcasted_to(%w[stream stream_2]).from_channel(channel).description).to eq("have broadcasted exactly 1 messages to broadcast:stream:stream_2")
```

Comment:
Does the channel affect the description in some way?

Reply:
It's required because it can't be implicitly inferred by the matcher. It's not included in the failure message, so I didn't include it in the implicit description either. I think it might be a useful thing to include in both, though?